test/api/transport/transport.transactions.main.js

```diff
@@ -1,257 +0,0 @@
-'use strict';
```

Comment:
@diego-G: The initial plan was to test the basic functionality of type 0 over webSocket protocol.